README.md

```diff
@@ -1,141 +1,146 @@
-
 # Aragon Network
 
-***Abstract***
-An opt-in jurisdiction that facilitates more efficient interactions between blockchain-native digital entities using economic incentives. Users are required to stake digital assets in order to participate in governance and to collateralize subjective agreements. In the event of a dispute a decentralized court serves as an oracle to resolve and enforce agreements between participants. An algorithmic monetary policy manages reserves and issuance of ANT to incentivize participation and healthy growth of the network.
+The Aragon Network is an Aragon organization which provides infrastructure and services to users of the Aragon platform, and is governed by ANT holders. The existing Aragon infrastructure enables users to create and manage organizations. Each Aragon organization exists as a set of smart contracts which define the organization's stakeholders and their associated rights and privileges. However, some rights and privileges require subjective constraints which cannot be encoded in a smart contract directly.
+
+The Aragon Court is a decentralized oracle protocol developed and maintained by the Aragon Network. The Aragon Court can be used by organizations, including the Aragon Network, to resolve subjective disputes with binary outcomes. When combined with the existing Aragon infrastructure, it enables an organization to create *proposal agreements* which define subjective constraints on an organization's operation that can be enforced by a minority stakeholders.
+
+## Aragon's Permission Architecture
+
+Aragon Organizations control which addresses have access to perform actions on behalf of the organization in a permission registry called the *Access Control List*. Addresses on the registry can be externally owned accounts or contracts. Some contracts are intended to *forward* actions based on pre-defined criteria, for example a voting app will forward action after the successful approval vote.
+
+By chaining multiple contracts together we can define complex criteria which constrain how actions can be performed within the organization. To illustrate this we can look at common scenario where an organization want to allow treasury funds to be transferred, but only if they are 1) proposed by a member of the organization, 2) approved by a majority of members, and 3) within a pre-determined budget. This can be accomplished by configuring a chain of permissions with each link imposing logical constraints on the final action as follows:
```

Comment:
```suggestion
By chaining multiple contracts together we can define complex criteria which constrain how actions can be performed within the organization. To illustrate this we can look at common scenario where an organization wants to allow treasury funds to be transferred, but only if they are 1) proposed by a member of the organization, 2) approved by a majority of members, and 3) within a pre-determined budget. This can be accomplished by configuring a chain of permissions with each link imposing logical constraints on the final action as follows:
```